packages/ListBox/src/components/Divider/Divider.styles.js

```diff
@@ -2,41 +2,35 @@ import styled, { css } from "styled-components";
 import tokens from "@paprika/tokens";
 import stylers from "@paprika/stylers";
 
-const pseudoElement = css`
-  background: ${tokens.color.blackLighten60};
-  content: "";
-  display: inline-block;
-  flex-grow: 1;
-  height: 1px;
-`;
+/*
+ * IMPORTANT: THE STYLE FOR ALL OPTION MUST BE THE SAME SIZE
+ * THIS IS IMPORTANT FOR CALCULATING THE CORRECT POSITION WHEN EXECUTING useOnScrolled() on Options/helpers/options.js
+ * IF THIS STYLES CHANGES THEN UP AN DOWN WILL NOT WORK CORRECTLY WHEN THE DIVIDER EXISTS
+ * PAY ATTENTION WHEN YOU MODIFY THE FOLLOWING ATTRIBUTES:
+ * - border
+ * - margin
+ * - padding
+ */
 
-const textDividerStyles = css`
-  &:after {
-    ${pseudoElement}
-    margin-left: ${tokens.space};
-  }
-`;
+export const Divider = styled.li(
+  ({ isDisabled }) => css`
```

Comment:
This will now dim when the (inline) component is disabled.

Reply:
/*
 * IMPORTANT: THE STYLE FOR ALL OPTION MUST BE THE SAME SIZE
 * THIS IS IMPORTANT FOR CALCULATING THE CORRECT POSITION WHEN EXECUTING useOnScrolled() on Options/helpers/options.js
 * IF THIS STYLES CHANGES THEN UP AN DOWN WILL NOT WORK CORRECTLY WHEN THE DIVIDER EXISTS
 * PAY ATTENTION WHEN YOU MODIFY THE FOLLOWING ATTRIBUTES:
 * - border
 * - margin
 * - padding
 */


I don't think this is applicable anymore since @oscarkwan change the implementation to focus on the option.

before the implementation controlled and scroll the container to the next options, now we don't need to the focus do it by itself. As far I can think of.